helm-chart/renku/templates/notebooks/env-secret.yaml

```diff
@@ -30,6 +30,8 @@ stringData:
   NB_GIT__REGISTRY: {{ required "An image registry must be specified." .Values.global.gitlab.registry.host }}
   NB_SESSIONS__GIT_RPC_SERVER__IMAGE: "{{ .Values.notebooks.gitRpcServer.image.name }}:{{ .Values.notebooks.gitRpcServer.image.tag }}"
   NB_SESSIONS__GIT_PROXY__IMAGE: "{{ .Values.notebooks.gitHttpsProxy.image.name }}:{{ .Values.notebooks.gitHttpsProxy.image.tag }}"
+  NB_SESSIONS__GIT_PROXY__PORT: "{{ .Values.notebooks.gitHttpsProxy.port | default 65480 }}"
+  NB_SESSIONS__GIT_PROXY__HEALTH_PORT: "{{ .Values.notebooks.gitHttpsProxy.healthPort | default (add (.Values.notebooks.gitHttpsProxy.port | default 65480) 1) }}"
```

Comment:
Just set the default in the values file. That is easier to figure out than if they are in the helm templates.

Reply:
I know we mention the new values file in the changelog but it is much clearer when you can also see the new key and default value in the values file.

Reply:
Done in 20eec085.